src/PrestaShopBundle/Resources/config/services/bundle/form/form.yml

```diff
@@ -1,9 +1,11 @@
 services:
   _defaults:
-    public: true
+    public: false
+    autowire: true
+    autoconfigure: true
 
-  form.form_cloner:
```

Comment:
I think you need to keep this service definition as deprecated for backward compatibility We'll be able to clean/remove it in 9.0